src/main/java/core/basesyntax/service/impl/FruitServiceImpl.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.FruitService;
+import core.basesyntax.strategy.FruitStrategy;
+import core.basesyntax.strategy.OperationsStrategy;
+import java.util.List;
+
+public class FruitServiceImpl implements FruitService {
+    @Override
+    public void getAllOperationsStrategy(List<FruitTransaction> fruitFromCsvRow,
```

Comment:
let's rename this method parameter in order to make it more clear. Also let's pass fruitStrategy to the constructor instead of method parameter. Agree with @pkobevko